docs/HowTo/Get-Started/Run-Docker-Image.md

```diff
@@ -0,0 +1,107 @@
+---
+description: Run Teku using the official docker image
+---
+
+# Running Teku from a Docker image
+
+Teku provides a Docker image to run a Teku node in a Docker container.
+
+Use this Docker image to run a Teku node without installing Teku.
+
+**Prerequisites**:
+
+* [Docker](https://docs.docker.com/install/)
+
+## Run Teku Docker image
+
+Display the Teku command line help using the Docker image
+
+```bash
+docker run pegasyseng/teku:develop --help
+```
+
+You can specify
+[Teku environment variables](../../Reference/CLI/CLI-Syntax.md#teku-environment-variables) with the
+docker image instead of the command line options.
+
+!!! example
+
+    ```bash
+    docker run -p 9000:9000 -p 5051:5051 -e TEKU_REST_API_ENABLED=true -e TEKU_P2P_PORT=9000 -e TEKU_LOG_FILE=/var/lib/teku/LOG --mount type=bind,source=/Users/user1/teku/,target=/var/lib/teku pegasyseng/teku:develop --eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd --eth1-endpoint=http://102.10.10.1:8545 --validators-key-file=var/lib/teku/validator_keys.yaml
+    ```
+
+
+## Exposing ports
+
+Expose ports for P2P peer discovery, metrics, and REST APIs. You need
+to expose the ports to use the default ports or the ports specified using
```

Comment:
Expose the default ports or the ports specified using: 

Suggested edit to make active

Reply:
Done